src/index.ts

```diff
@@ -26,6 +26,11 @@ export { LinkTypes } from "./components/Link/LinkTypes";
 export { default as List } from "./components/List/List";
 export { ListTypes } from "./components/List/ListTypes";
 export { default as Modal } from "./components/Modal/Modal";
+// export {
```

Comment:
Is this ready to be exported or why is this commented out?

Reply:
Yes, an oversight.  Fixed.